community/gill-supabase-web3/prisma/schema.prisma

```diff
@@ -0,0 +1,21 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id             String   @id @default(cuid())
+  walletAddress  String   @unique @map("wallet_address")
+  supabaseUserId String?  @map("supabase_user_id")
```

Comment:
We are saving the user here to have more metadata than are provided by 

signInWithWeb3 methods?